conf/messages.en

```diff
@@ -396,7 +396,8 @@ schemeDetails.checkYourAnswersLabel = Scheme details
 schemeDetails.error.required = Enter Scheme details
 schemeDetails.error.name.length = Scheme-name must be 150 characters or less
 schemeDetails.error.ref.length = Reference must be 150 characters or less
-schemeDetails.change.hidden = Scheme details
+schemeName.change.hidden = Scheme name
```

Comment:
do we need this ?

Reply:
we should have a call, schemeDetails.change.hidden is still there as hidden text without message value (SchemeDetailsSummary. Something seems wrong here